lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -382,6 +469,11 @@ def cmd_hosts(*args)
             when "svcs";      host.services.length
             when "vulns";     host.vulns.length
             when "workspace"; host.workspace.name
+            when "tags"
+              found_tags = Mdm::Tag.includes(:hosts).where("hosts.workspace_id = ? and hosts.address = ?", framework.db.workspace.id, host.address).order("tags.id DESC")
+              tag_names = []
+              found_tags.each {|t| tag_names << t.name}
```

Comment:
Can you pad the braces like in https://github.com/rapid7/metasploit-framework/pull/4982/files#diff-2cbd0086dda03d12d7a84d971eddadbaR460? Minor nitpick. Thanks.